service/systemd/service.go

```diff
@@ -582,3 +595,8 @@ func (s *Service) WriteService() error {
 	}
 	return nil
 }
+
+// SysdReload reloads Service daemon.
+func SysdReload() {
```

Comment:
Sorry I didn't notice this before, but this doesn't actually reload the daemon - it just builds the command string to do it. To reload the daemon you'd need to either add a reload method to Cmdline to call `runCommand` with the command returned by `cmds.reload()`, or call the `DBusAPI.Reload`.